tests/pref_improvement/pref_check.py

```diff
@@ -0,0 +1,105 @@
+import inspect
+import importlib
+import sys
+from typing import Iterable
+from dataclasses import is_dataclass, fields
+from pathlib import Path
+
+import csvcubed
+from csvcubed.definitions import APP_ROOT_DIR_PATH
+from csvcubed.models.validatedmodel import ValidatedModel
+
+# scan throught each file and check if the members are classes
```

Comment:
These comments should be docstring comments below the function's definition. This way the information will show up in your IDE's intellisense.